triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -57,14 +57,14 @@ final class Converter(
   private[this] def translateValue(ty: Type, value: Value): Either[String, SValue] =
     valueTranslator.translateValue(ty, value).left.map(res => s"Failure to translate value: $res")
 
+  private[this] val triggerIds: TriggerIds = triggerDef.triggerIds
+
   private[this] val templateTypeRepTyCon = DA.Internal.Any.assertIdentifier("TemplateTypeRep")
   private[this] val anyTemplateTyCon = DA.Internal.Any.assertIdentifier("AnyTemplate")
   // TODO: https://github.com/digital-asset/daml/issues/14830
   // replace by DA.Internal.Any.AnyView.AnyView once it is introduced as stable package
   private[this] val anyViewTyCon = DA.Internal.Any.assertIdentifier("AnyView")
```

Comment:
```suggestion
  private[this] val templateTypeRepTyCon = DA.Internal.Any.TemplateTypeRep
  private[this] val anyTemplateTyCon = DA.Internal.Any.AnyTemplate
  private[this] val anyViewTyCon = DA.Internal.Interface.AnyView.Types.AnyView
```
with 
```
val AnyView: TypeConName = assertIdentifier("AnyView")
```
in the body of `DA.Internal.Interface.AnyView.Types`

Reply:
I would probably inline those values as they are used only once.